commcare_connect/opportunity/views.py

```diff
@@ -1396,6 +1399,11 @@ def user_visit_verification(request, org_slug, opp_id, pk):
     flagged_info = defaultdict(lambda: {"name": "", "approved": 0, "pending": 0, "rejected": 0})
     for visit in visits:
         for flag in visit.flags:
+            if flag == "form_submission_period":
```

Comment:
These need to exist as some sort of Dict constants somewhere, right?

Reply:
Yeah, these need to be taken out into a constant for easier reference in future. I can add that change in the form processor changes that I will work on later.